android/src/main/java/com/saranshmalik/rnzendeskchat/RNZendeskChat.java

```diff
@@ -165,18 +166,30 @@ public void setUserIdentity(ReadableMap options) {
     checkIdentity();
   }
 
+  @ReactMethod
+  public void openTicket(){
+      Activity activity = getCurrentActivity();
+
+      List<CustomField> customFields = new ArrayList<>();
+      // Custom Field for the category
+      customFields.add(new CustomField(1900002821093L, "Cashback"));
```

Comment:
Is this needed?

Reply:
@debiff should we also increase the version of the package?

Reply:
> 

No, it's a test I forgot to remove